### PR TITLE
Add buff parameter to BraceLabel

### PR DESCRIPTION
**Pull request by sparshg**
## Overview: What does this pull request change?

Quickfix for #1769 

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

File: manim/mobject/svg/brace.py [hunk 1]
```diff
@@ -140,16 +140,18 @@ def __init__(
...
         label_scale=1,
+        buff=0.2,
         **kwargs
     ):
         self.label_constructor = label_constructor
         self.label_scale = label_scale
         super().__init__(**kwargs)
 
         self.brace_direction = brace_direction
+        self.buff = buff
```
Comment (ryanmccauley211) [inline manim/mobject/svg/brace.py:151]: Do we need to store it here? Seems like it is just passed straight through to Brace

**Comment by ryanmccauley211** [comment]
Looks good! Works for me locally, just left a minor comment

File: manim/mobject/svg/brace.py (hunk 1 above)
Comment (sparshg) [reply manim/mobject/svg/brace.py:151]: ye not needed but added that just in case for future, idk

**Review by ryanmccauley211 — APPROVED** [review]

**Review by hydrobeam — APPROVED** [review]
LGTM

**Pull request merged**